interface/super/edit_list.php

```diff
@@ -618,9 +618,11 @@ function writeITLine($it_array) {
 <?php html_header_show();?>
 
 <!-- supporting javascript code -->
-<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/jquery.js"></script>
-
+<script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/js/jquery-3.3.1.min.js"></script>
```

Comment:
`require_once $GLOBALS['srcdir'].'/headers.inc.php'; ` is on line 22 of this file.  

To call a single library from the assets/js/ directory, use `function include_js_library($path)` where $path is the directory name of the asset under assets/js/ and the actual file name is index.js.

`function include_css_library($path)` is the same as the above, just requires the full file path.

`call_required_libraries($library_array)` allows multiple libraries to be called.  In your case, you would invoke it with `call_required_libraries('jquery-min-3-3-1','select2');`

The above requires renaming the files to index.js, and requires that the directory naming standard be put into the jquery-min-* naming standard.  Either a default case switch or a final elseif else statement needs to be added that invokes whatever path is passed in the argument array, OR add "select2" to the list of 'if' conditions.

These functions require some rework but when we change the functions themselves to make them more efficient (lots of people touched the file) and provide default vs. specified behavior of include_js_library($path) to be added to call_required_libraries , but we should not have to ever change the calls.